cobalt/black_box_tests/black_box_tests.py

```diff
@@ -194,8 +194,12 @@ def LoadTests(launcher_params, test_set):
 
   test_suite = unittest.TestSuite()
   for test in test_targets:
-    test_suite.addTest(unittest.TestLoader().loadTestsFromModule(
-        importlib.import_module(_TEST_DIR_PATH + test)))
+    for filtered_test in test_filters:
+      if test_set == filtered_test.target_name and test == filtered_test.test_name:
+        continue
+      else:
```

Comment:
nit: unnecessary

Reply:
Specifically the `else:` is unnecessary, you already continue the loop in the above conditional

Reply:
That "continue" breaks the loop that goes through the filtered tests. It is actually a bug.